db_router.py

```diff
@@ -0,0 +1,34 @@
+
+
+class TmpBaseRouter(object):
+    """
+    A router to control all database operations on models in the
+    auth application.
+    """
+    def db_for_read(self, model, **hints):
+        """
+        Attempts to read auth models go to auth_db.
+        """
+        if model._meta.model_name == 'search':
+            return 'tmp'
+        return None
+
+    def db_for_write(self, model, **hints):
+        """
+        Attempts to write auth models go to auth_db.
+        """
+        if model._meta.model_name == 'search':
+            return 'tmp'
+        return None
+
+    def allow_relation(self, obj1, obj2, **hints):
+        """
+        Allow relations if a model in the auth app is involved.
+        """
+        if obj1._meta.model_name == 'search' or \
+           obj2._meta.model_name == 'search':
+            return True
+        return None
+
+    def allow_migrate(self, db, app_label, model_name=None, **hints):
```

Comment:
"we do not care" is "use the default behavior".
It can depend on a lot of things (other Database Routers) but, it is probably True in our case.
However we can indeed set it to True be sure that migration will be done.